### ISSUE TITLE
lib64 symlink can break subtly since 16.1

### ISSUE THREAD

**Issue by wouterdb** [pypa/virtualenv#1248]
## tldr
When creating a virtualenv on a path that contains a symlink, the virtualenv break if this symlink is removed due to the changes made in https://github.com/pypa/virtualenv/commit/5d560144432642a91f6e1661bed22e637226c8fe

 ## context
When running tests, we create a workspace folder for each test, which has a symlink to a shared virtual env. The first test case populates the virtualenv. Then the first testcase's workspace is removed. 

Prior to 16.1 this worked as expected. However, in 16.1 the lib64 symlink is absolute (but not canonical). So when the workspace of the first test is removed, the shared virtual env breaks. The next test run produces the error "module pip not found". 

## fix 
revert the change on line 1622 (https://github.com/pypa/virtualenv/commit/5d560144432642a91f6e1661bed22e637226c8fe#diff-51642b24ac9529f4baa0f483fa257d2dL1622)
